.claude/skills/cmux-codex-autoapprove/scripts/cmux_codex_approve_launchd.py

```
@@ -0,0 +1,738 @@
+            ):
+                latest_match = window_text
+                break
+    return latest_match
+
+
+def heuristic_decision(tail_text: str) -> str:
+    if SELECTED_OPTION_1_RE.search(tail_text) and ENTER_SUBMIT_RE.search(tail_text):
+        return "ENTER"
+    if EDIT_PROMPT_RE.search(tail_text) and SELECTED_OPTION_1_RE.search(tail_text) and ESC_TAB_AMEND_RE.search(tail_text):
+        return "ENTER"
+    # Cursor is already on option 1 (❯ 1.) and that option is Yes/Allow — press Enter.
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `❯` (HEAVY RIGHT-POINTING ANGLE QUOTATION MARK ORNAMENT). Did you mean `>` (GREATER-THAN SIGN)?

(RUF003)